### PR TITLE
Updating guest bootstrap.

### PR DESCRIPTION
**Pull request by hub-cap**
- Moved the files that are used in guest to files/
- Modified redstack to use homedir for image creation
- Modified redstack to mv the files to tmpdir and sed them for use in vm

### CODE REVIEW

File: scripts/redstack [hunk 1]
```diff
@@ -181,6 +181,7 @@ REDDWARF_ROLE=$REDDWARF_ROLE
     SERVICE_UUID=`keystone_manage service-list | grep reddwarf | awk '{print $2}'`
     if [ -z $SERVICE_UUID ]; then
         OS_USERNAME=reddwarf OS_PASSWORD=REDDWARF-PASS \
+        OS_AUTH_URL='http://localhost:8779/v0.1/$(tenant_id)s' \
```
Comment (hub-cap) [inline scripts/redstack:184]: i have NO clue if this is the right URL to put, just that it fixes the issue w/ initialize.

File: scripts/redstack [hunk 2]
```diff
@@ -547,7 +552,7 @@ cmd_build_image() {
 
     USERNAME=`whoami`
     HOMEDIR=`getent passwd $USERNAME | cut -d: -f6`
-    VM_PATH=/tmp/oneiric_mysql_image
+    VM_PATH=/home/$USERNAME/oneiric_mysql_image
```
Comment (hub-cap) [inline scripts/redstack:555]: Everyone can rejoice. Now when the host VM is rebooted you wont lose your built image.

**Comment by hub-cap** [comment]
please note, with the changes to this, you should consider rebuilding your image!

**Comment by cp16net** [comment]
+1

File: scripts/redstack (hunk 1 above)
Comment (rnirmal) [inline scripts/redstack:184]: If you have the endpoint and token you shouldn't need OS_USERNAME/PASSWORD or OS_AUTH_URL

Comment (hub-cap) [inline scripts/redstack:184]: You tell the command that. It keeps on throwing up w/o it and it tells me I need it. :)

**Comment by TimSimpsonR** [comment]
+1 and I want to merge this nows so I can use it.
However I should point out the files directory may need to have a directory specified as "guest" since all those files are used to build the guest image.

**Comment by rnirmal** [comment]
+1

File: scripts/redstack (hunk 1 above)
Comment (rnirmal) [inline scripts/redstack:184]: the usual wonkiness with any of the nova clients...... You need a degree in "Rocket Science" to figure it out :)

**Pull request merged**